modules/ROOT/pages/ocis_release_notes.adoc

```diff
@@ -15,6 +15,183 @@
 
 toc::[]
 
+== Infinite Scale 6.0.0
+
+[discrete]
+=== General
+
+IMPORTANT: This is a Rolling Release. Please check the https://owncloud.dev/ocis/release_roadmap/#release-types[documentation] to see if this release type is right for your use case. There is no upgrade path from previous releases, so this version is therefore only suitable for you if you start with a clean slate.
+
+Refer to the source and the full change log for a list of bug fixes and changes at {ocis-releases-url}/v6.0.0[GitHub, window=_blank].
+
+[discrete]
+=== Rolling Release
+
+This is the first https://owncloud.dev/ocis/release_roadmap/#release-types[Rolling Release] of Infinite Scale.
+
+In addition to our Production releases, you can now install the new release type *Rolling*, which allows you to experience the latest features without having to wait for a Production release. The Rolling release offers access to the latest features and improvements every three weeks. This new release type complements our existing Production and Daily releases, providing a flexible and dynamic update cycle perfect for early adopters and enthusiasts.
+
+[discrete]
+#### How to Get the Rolling Release
+
+[discrete]
+##### Docker
+
+The images available on https://hub.docker.com[Docker Hub] are now separated to clearly distinguish between *production* and *rolling*.
+
+* *Production* +
+is located at https://hub.docker.com/r/owncloud/ocis and can be used via:
++
+[source,shell]
+----
+docker pull owncloud/ocic:latest
+----
+
+* *Rolling* +
+is located at https://hub.docker.com/r/owncloud/ocis and can be used via:
++
+[source,shell]
+----
+docker pull owncloud/ocis-rolling:latest
+----
+
+[discrete]
+#### Binary
+
+The binary files to download are now separated to clearly distinguish between production and rolling.
+
+* *Production* +
+can be downloaded from: https://download.owncloud.com/ocis/ocis/stable/[, window=_blank]
+
+* *Rolling* +
+can be downloaded from: https://download.owncloud.com/ocis/ocis/rolling/[, window=_blank]
+
+[discrete]
+=== Web Office: Collabora Secure View
+
+Infinite Scale now supports Collabora’s Secure View feature, enhancing security for sensitive documents by allowing restricted viewing. Recipients can view content without downloading, copying, or editing, and a watermark with the user’s name is always applied to prevent unauthorized sharing. Secure View supports for example office documents, images, and PDFs. Read all about https://www.collaboraoffice.com/security/collabora-secure-view/[Collabora`s Secure View]. Collabora is the favored web office application of Infinite Scale, known for its strong focus on secure collaboration, making it ideal for organizations prioritizing data security and privacy.
```

Comment:
```suggestion
Infinite Scale now supports Collabora’s Secure View feature, enhancing security for sensitive documents by allowing restricted viewing. Recipients can view content without downloading, copying, or editing, and a watermark with the user’s name is always applied to prevent unauthorized sharing. Secure View supports, for example, office documents, images, and PDFs. Read all about https://www.collaboraoffice.com/security/collabora-secure-view/[Collabora`s Secure View]. Collabora is the favored web office application of Infinite Scale, known for its strong focus on secure collaboration, making it ideal for organizations prioritizing data security and privacy.
```

And I wonder what are the best words for "to prevent unauthorized sharing" - the user can still take a local copy of the "bitmap" that is rendered in the browser, with the watermark, and then share it wherever they like. The watermark just means that the user can't do it anonymously. (or would have to do some image-processing to remove the watermark)